tests/pkcli/argh_test.py

```diff
@@ -0,0 +1,22 @@
+# -*- coding: utf-8 -*-
+"""pytest for `pykern.pkcli.argh_test`
```

Comment:
We don't need to mention pytest. It is a test for pkcli arg parsing.

Reply:
7ff4012